shimmingtoolbox/cli/st_mask.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python3
+
+import logging
+import click
+import numpy as np
+
+from shimmingtoolbox.masking import threshold
+from shimmingtoolbox.masking import shapes
+# import SCT
+# import BET
+
+@click.command()
+@click.argument("input", required=True, type=click.File('rb')) # 1 file à la fois ou bien plusieurs d'un coup avec nargs=-1 ?
+@click.option("--method", required=True, type=click.Choice(['SCT','BET','shape','threshold'], case_sensitive=False), help="Indicate which kind od mask: SCT, BET, shape or threshold")
+@click.option("--shape", type=click.Choice(['cube','square'], case_sensitive=False), default='cube', help="Indcate which kinf of shape for method=shape: cube or square")
```

Comment:
PEP8 limit to how long a line is.

```suggestion
@click.option("--shape", type=click.Choice(['cube','square'], case_sensitive=False), default='cube', 
              help="Indcate which kinf of shape for method=shape: cube or square")
```